### PR TITLE
Vet clinic: create animals table

### PR DESCRIPTION
**Pull request by Sonickmumba**
In this pull request, I 

- Created the database vet_clinic
- Created a table named `animals` with the following columns and copied the SQL to the `schema.sql` file:
  - `id`: integer
  - `name`: string
  - `date_of_birth`: date
  - `escape_attempts`: integer
  - `neutered`: boolean
  - `weight_kg`: decimal
- Inserted the following data copied the SQL to the `data.sql` file:
  - Animal: His name is Agumon. He was born on Feb 3rd, 2020, and currently weighs 10.23kg. He was neutered and he has never tried to escape.
  - Animal: Her name is Gabumon. She was born on Nov 15th, 2018, and currently weighs 8kg. She is neutered and she has tried to escape 2 times.
  - Animal: His name is Pikachu. He was born on Jan 7th, 2021, and currently weighs 15.04kg. He was not neutered and he has tried to escape once.
  - Animal: Her name is Devimon. She was born on May 12th, 2017, and currently weighs 11kg. She is neutered and she has tried to escape 5 times.
- Wrote the following queries for the quests with desired results:
  - Find all animals whose name ends in "mon".
  endsWith-mon

  - List the name of all animals born between 2016 and 2019.
  date_of_birth

  - List the name of all animals that are neutered and have less than 3 escape attempts.
  neutered-and-less-than3Escape

  - List the date of birth of all animals named either "Agumon" or "Pikachu".
  named-Agumon or Pikachu

  - List name and escape attempts of animals that weigh more than 10.5kg
  more-than-10kg

  - Find all animals that are neutered.
  neutered-animals

  - Find all animals not named Gabumon.
  name-not-Gabumon

  - Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
  between-10-17

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,12 @@
 /* Database schema to keep the structure of entire database. */
 
+CREATE DATABASE vet_clinic;
+
 CREATE TABLE animals (
-    name varchar(100)
+    id integer,
```
Comment (sja-thedude) [inline schema.sql]: - [ ] Optional: You can make id column as Not Null and add PRIMARY KEY to it.

File: data.sql [hunk 2]
```diff
@@ -1,5 +1,6 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (1, 'Agumon', '2020-02-03', 0, 'Yes', 10.23);
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (2, 'Gabumon', '2018-11-15', 2, 'Yes', 8);
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (3, 'Pikachu', '2021-01-17', 1, 'No', 15.04);
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (4, 'Devimon', '2017-05-12', 5, 'Yes', 11);
```
Comment (sja-thedude) [inline data.sql]: - [ ] It would be better if you use True and False instead of Yes and No for neutered values.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @Sonickmumba, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Professional Readme
✔️ Correct Gitflow

## Required  
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 2 above)
Comment (Sonickmumba) [reply data.sql]: Noted with thanks

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @Sonickmumba , 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 

✅ Descriptive PR 👍 
✅ Nice schema 🔥 
✅ Linters are passing 👏 


## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**